bin/mock_segment.pl

```diff
@@ -0,0 +1,36 @@
+# Automatically enables "strict", "warnings", "utf8" and Perl 5.10 features
+use Mojolicious::Lite;
+use Syntax::Keyword::Try;
+
+################################################################################
```

Comment:
Do you want to put `Mojolicious::Lite` in dependencies only for tests? maybe better just mock/stub http client, you don't need to test http client.

Reply:
@kirill-binary  I've mocked `Net::Async::HTTP`->POST in the test script. But it is too do the change you asked with it, because it is not easy to generate HTTP response object as return value. Which method/object do you recommend as an easier alternative?

Reply:
you don't have to generate whole HTTP response, you could just create a stub with required methods for example:
```
my $ua = Test::MockObject->new();
$ua->mock(do_request => sub {
    my $response = Test::MockObject->new();
    $response->mock(content => sub {'Some non JSON response'});
    Future->done($response);
});
```

Reply:
Done. Thank you.